# Oat pangenome databases
This section include the Introduction and Tutorials for the Oat pangenome tools available on the database.

## <span style="color:#023047"> I. Introduction </span>

The Oat databases includes:  
* Genome browsers with high quality assemblies and annotations of 26 Oat accessions.  
* Pan-transcriptome data of 26 Oat accessions.  

### 1. Project outcomes



### 2. Project funding and extra information

## <span style="color:#023047"> II. Current available data </span>

### 1. Genome assembly and annotation

Including genome assemblies and annotations from 26 oat accessions. Detailed accession list can be seen [here](/info/oat_pangenome_accessions.md)

### 2. Genetic variants

Genetic variants including InDels and SNPs were obtained from a wide range of datasets, including:  
* High coverage - whole genome sequence of [564 oat accessions]() from a world-wide collection.  
* High coverage - whole genome sequence of [90 oat accessions]() highly related to Australian oat breeding programs.  

### 3. Pan-transcriptome data

The Oat Pan-transcriptome database is obtained 6 different tissues of 26 Oat accessions from the pangenome collection.  


## <span style="color:#023047"> III. Tools overview </span>

### 1. Getting start

The WCGA Oat pangenome database can be accessed using this [link](http://database.oatpangenome.com)  
To start using the WCGA Oat pangenome database, you need to register an account. Please refer to this [link](tools/registration.md) for information about the registration process.  
For more information about each tool and available data, please refer to the below sections.  

### 2. Oat genome browser  

For a detailed instruction about using the Barley genome browser, please refer to this link: [The Oat genome browsers](tools/oat_jbrowse.md)

The Oat genome browsers were built based on [Jbrowse v2](https://jbrowse.org/jb2/) with whole genome sequences and annotations of 26 Oat accessions from the pangenome collection.

#### a. Features

* Genes lookup by gene IDs  
* Gene and CDS sequences extraction, including up stream and down stream regions  
* Cross-variant gene ID search. Search using Bannister and GS7 gene IDs on OT3098v2 positions, gene IDs matching.
* Genetic variants view with annotations  

#### b. Data currently available on the genome browsers

**Genome assembly tracks**

* Genome assemblies of 26 oat accession from the [pangenome collection](/info/oat_pangenome_accessions.md)   

**Annotations tracks**

* Functional annotations for 26 barley accession of the [pangenome collection](/info/oat_pangenome_accessions.md)  
* Bannister and GS7 lift off to OT3098v2 for quick gene ID convert (reference: OT3098v2 assembly)  

**Variant tracks - OT3098v2 reference**

* Diversity Array Technology Sequence (DArT-seq) for [1,000 barley accessions](https://ftp.ebi.ac.uk/biostudies/fire/E-MTAB-/362/E-MTAB-7362/Files/E-MTAB-7362.sdrf.txt)

## 3. Oat variant database

For a detailed instruction about using the Barley variant database, please refer to this link: [The Oat variant databases](tools/oat_variantdb.md)

#### a. Features

* Query using physical position or gene IDs  
* Extraction of genetic information around the variants for primers designing  
* Result table export to csv format  

#### b. Data currently available

[The Barley variant databases](tools/barley_variantdb.md) contains approximately 54 million SNPs and 3.6 million InDels, from a collection of 21 Barley accessions.
These variants were called based on Morex v1 sequence and annotated using Morex v1 annotation.

### 4. Oat Pan-transcriptome database

Detailed instruction about using the [Barley Linear Pan-transcriptome database](tools/barley_panbart20.md)

#### a. Features

* Query for gene expression using Morex V3 gene IDs    
* Heatmap of gene expression of 5 different tissues from 20 Barley accessions

#### b. Data currently available


## <span style="color:#023047"> IV. Work in progress </span>
