# Barley variant database tutorials

### 1. The variant database can be accessed using the following link

http://database.barleypangenome.com/search/

Or click on Database on the navigation bar on top of the website

<img src="img/barleydb/00_start.png" alt="hi" width="800" class="inline"/>

**Select Barleyvar from the database list**

<img src="img/barleydb/01_db_select.png" alt="hi" width="800" class="inline"/>


### 2. Query by physical positions

**For physical position query, different format can be used**

* Chromosome:start-end (chr1H:123-456)  
* Chromosome:position to query 2kb up and downstream (chr1H:123456)  
* Chromosome:start+length to query an exact region from start position (chr1H:123+500)  

<img src="img/barleydb/02_search_position.png" alt="hi" width="800" class="inline"/>

### 3. The result is display in a table format

<img src="img/barleydb/03_result.png" alt="hi" width="800" class="inline"/>

### 4. The sequence around the variant can be retrieve using the sequence icon next to the variant ID

*The sequence is pre-formated in fasta format and the variant is in red*  

<img src="img/barleydb/04_sequence.png" alt="hi" width="800" class="inline"/>

### 5. The whole result table can be exported into csv format

<img src="img/barleydb/05_export.png" alt="hi" width="800" class="inline"/>

## 6. SNPs and annotations

**By default, only Indels is pre-selected. SNPs and variant annotations data can be enable using the check boxes**

<img src="img/barleydb/06_SNP_select.png" alt="hi" width="800" class="inline"/>

## 7. Query by geneIDs

**All variants within the gene regions can be query using gene IDs**

*The example gene ID format is HORVU3Hr1G090980*  

<img src="img/barleydb/07_geneid_search.png" alt="hi" width="800" class="inline"/>

## 8. Distinguish SNPs and Indels from the result

**Indels and SNPs (if enable) can be distinguish using their variants ID on the left column**

<img src="img/barleydb/08_SNP_Indels.png" alt="hi" width="800" class="inline"/>
